Goof around

Patater opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/alzix/try-reviewable-io/3)
<!-- Reviewable:end -->

Patater committed:
Add some junk

Let's play around with junk. It's fun! I hope.

Patater committed:
XXX remove trailing whitespace

This should be applied as a fixup commit later.

Patater committed:
XXX Actually remove all trailing whitespace

Another fixup commit!

alzix commented:
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions.

---

*[main.c, line 10 at r1](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3#-KtBL5tLn8DA60fLbhHO:-KtBL5tLn8DA60fLbhHP:b-524sd7) ([raw file](https://github.com/alzix/try-reviewable-io/blob/809140f2f3aa6920a25fc7c2deee0f3e81c43bf0/main.c#L10)):*
> ```C
> 
> int banana = 0x00BA7A7A;
> extern int ding_dong;
> ```

who defines it?

---

*[README.md, line 4 at r1](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3#-KtBKxXW0DUHBU1mcNED:-KtBKxXW0DUHBU1mcNEE:b-j07utr) ([raw file](https://github.com/alzix/try-reviewable-io/blob/809140f2f3aa6920a25fc7c2deee0f3e81c43bf0/README.md#L4)):*
> ```Markdown
> foofoo
> 
> How do you like this trailing whitespace?
> ```

seems it is just ignores it... :(

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3)*
<!-- Sent from Reviewable.io -->

alzix commented:
Reviewed 1 of 2 files at r1.
Review status: 1 of 2 files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/alzix/try-reviewable-io/3)*
<!-- Sent from Reviewable.io -->

Patater committed:
XXX add comment about where ding_dong is from

To be applied as fixup

Patater commented:
> seems it is just ignores it... :(


Well, maybe. I have added it in the first commit, and fixed it in a later commit. I was hoping to be able to see the problem in an earlier commit.

Patater commented on README.md:
```diff
@@ -1,5 +1,5 @@
 foofoo
 
-
+How do you like this trailing whitespace?   
```
This line has trailing whitespace.

Patater committed:
XXX Add some whitespace errors for fun

This commit should be discarded

Patater commented:
I'm done goofing around.
